### DISCUSSION TITLE
folder contents didnt move with folder

### DISCUSSION THREAD

**Discussion by cosmic8weave** [iterate-ch/cyberduck#15334] [Q&A]
hi :)

i just moved a folder into a higher level in the parent directory and the folder contents were not moved together with the folder ... what can i do to restore the file? ... thank you :)

**Comment by tatze96** [comment]
Which protocol is in use?
Are the files still visible on the server outside of Cyberduck or Mountain Duck?

Missing files are usually not recoverable unless caught within the server's trash folder or by some kind of versioning functionality.
